### ISSUE TITLE
Play of sound always return 0.

### ISSUE THREAD

**Issue by newMemberHub** [barsoosayque/libgdx-oboe#8]
Hi support team,
After updating the oboe in libgdx 1.9.11, I tried to play the sound and it can work fine but the return of play is always 0.
Ex: 
long ret1 = mySound.play(volume); // ret 1 is 0
long ret2 = mySound.play(volume); // ret 2 is 0
// both ret 1 and 2 are 0, they should be different.

**Comment by karivatj** [comment]
I, too am facing a similar issue. The sound plays fine but without the sound Id I cannot control the sound sample like setting pitch etc. Using libGDX 1.10.0. calling play() always returns zero. If I call for example setPitch by setting the sound id manually to 1 (first audio sample) the pitch changes. Theres something wrong with returning the id back to the caller.

**Comment by karivatj** [comment]
This issue is fixed in v 0.3.0. I compiled the library manually and imported the library to my project. Sound Id works fine now. Great!

Note: Is it possible to get this release to a maven repo so I can pull the version v 0.3.0 version from there instead of building it by hand?

But anyway, thanks again for the great work!

**Comment by barsoosayque** [comment]
Yeah, sorry I didn't reply about fix in this thread. I didn't release a new version because there is currently no place to upload release artifacts, sadly (since bintray's goodbye).

There is #10 . I tried to use github packages, but to no avail :(

**Comment by barsoosayque** [comment]
Let's keep this issue until I eventually find a place to host releases.

**Comment by Giuseppe1984** [comment]
Any news about this issue? I receive id=0 too :(

**Comment by karivatj** [comment]
The fix is there. You just have to build the library yourself. Like the author said, they couldn't upload the artifact because bintray was shutdown. I got it working by building the library and linking it to my project.
